docs/docsite/rst/playbook_guide/playbook_pathing.rst

```diff
@@ -21,11 +21,13 @@ Task paths
 
 Relative paths used in a task typically refer to remote files and directories on the managed nodes. However, paths passed to lookup plugins and some paths used in action plugins such as the "src" path for the :ansplugin:`template <ansible.builtin.template#module>` and :ansplugin:`copy <ansible.builtin.copy#module>` modules refer to local files and directories on the control node.
 
-Resolving local relative paths
-------------------------------
+
+Resolving local lookup and action plugin relative paths
```

Comment:
this is not limited to those cases, any plugin that attempts a file access when called from within a task will have this issue, which adds filter and test plugins to the mix also (see `exists` test for an example), also vars plugins.

This won't affect callbacks, inventory and many other plugins like that, which are more 'global' or execute independent of each task.